napalm_nxos/nxos.py

```diff
@@ -335,7 +339,8 @@ def cli(self, commands = None):
             try:
                 string_output = self.device.show(command, fmat = 'json', text = True)[1]
                 dict_output   = eval(string_output)
-                cli_output[unicode(command)] = self._get_reply_body(dict_output)
+                s = dict_output.get('ins_api', {}).get('outputs', {}).get('output', {}).get('body', '')
```

Comment:
Can we please have a less generic variabile name? Anyway some guidelines (not sure if pep8 or newer) suggest using at least 3 characters length.